src/components/LandingPage/DocsBtn.astro

```diff
@@ -5,11 +5,12 @@ type Props = {
 	classes?: string
 	href: string
 	title: string
+	glow?: boolean
 }
 
-const { text, href, isExternal, classes, title }: Props = Astro.props
+const { text, href, isExternal, classes, title, glow }: Props = Astro.props
 
-const css = `docs-btn ${classes ? classes : ''}`
+const css = `docs-btn ${glow ? 'glow' : ''} ${classes ? classes : ''}`
```

Comment:
You can shorthand it:
``${glow ?? ''`}``

Reply:
that would result in it returning true instead of desired string class name

Reply:
Not really: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing

In this case, it will return either "glow", which is a string or an empty string when the glow is undefined.

Reply:
![image](https://github.com/NowYouKnowProgramming/webdev-learning-materials/assets/55458485/ad79d566-86b3-474e-b0c8-69c4226f5424)

Reply:
Glow 
![image](https://github.com/NowYouKnowProgramming/webdev-learning-materials/assets/55458485/ffb55eba-7a3a-4aef-ac80-d5876e492ef0)

Reply:
Ah, my mistake.